source/containers.tex

```diff
@@ -6548,13 +6548,13 @@
 \begin{codeblock}
 template<class InputIterator>
   using @\placeholder{iter-value-type}@ =
-    typename iterator_traits<InputIterator>::value_type;                // \expos
+    iterator_traits<InputIterator>::value_type;                         // \expos
 template<class InputIterator>
   using @\placeholder{iter-key-type}@ = remove_const_t<
     typename iterator_traits<InputIterator>::value_type::first_type>;   // \expos
```

Comment:
http://wg21.link/p2165r3 will get rid of the other `typename`s in this code block.